src/main/java/org/jabref/logic/importer/fetcher/RfcFetcher.java

```diff
@@ -0,0 +1,53 @@
+package org.jabref.logic.importer.fetcher;
+
+import org.apache.http.client.utils.URIBuilder;
+import org.jabref.logic.help.HelpFile;
+import org.jabref.logic.importer.FetcherException;
+import org.jabref.logic.importer.IdBasedParserFetcher;
+import org.jabref.logic.importer.ImportFormatPreferences;
+import org.jabref.logic.importer.Parser;
+import org.jabref.logic.importer.fileformat.BibtexParser;
+import org.jabref.model.util.DummyFileUpdateMonitor;
+
+import java.net.MalformedURLException;
+import java.net.URISyntaxException;
+import java.net.URL;
+
+/*
+ * https://datatracker.ietf.org
+ * IETF (Internet Engineering Task Force) datatracker contains data about the documents,
+ * working groups, meetings, agendas, minutes, presentations, and more, of the IETF.
+ */
+public class RfcFetcher implements IdBasedParserFetcher {
+
+    private final ImportFormatPreferences importFormatPreferences;
+
+    public RfcFetcher (ImportFormatPreferences importFormatPreferences) {
+        this.importFormatPreferences = importFormatPreferences;
+    }
+
+    @Override
+    public String getName() {
+        return "RFC";
+    }
+    
+    @Override
+    public HelpFile getHelpPage() {
+        return HelpFile.FETCHER_RFC;
+    }
+
+    @Override
+    public URL getURLForID(String identifier) throws URISyntaxException, MalformedURLException, FetcherException {
+        // Add "rfc" prefix if user's search entry was only digits
+        identifier = (!identifier.startsWith("rfc")) ? "rfc" + identifier : identifier;
```

Comment:
Even better if you check for case insensitive with equals ignore case

Reply:
CodeCheck doesn't like this parameter reassigning... Ignore check? or suggestion?